spec/models/course_group_spec.rb

```diff
@@ -76,4 +76,15 @@
       end
     end
   end
+
+  describe "scopes" do
+    describe ".leadership_or_specialist" do
+      let!(:leadership_group) { CourseGroup.find_by(name: "leadership") || create(:course_group, name: "leadership") }
+      let!(:specialist_group) { CourseGroup.find_by(name: "specialist") || create(:course_group, name: "specialist") }
```

Comment:
Can we add a course group that isn't leadership/specialist to ensure its not returning?